docs/sdks-and-tools/en/rpc-endpoints.md

```diff
@@ -0,0 +1,55 @@
+---
+id: conflux_rpcs
+title: Public RPCs
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/sdks-and-tools/en/rpc-endpoints.md
+keywords:
+  - endpoints
+---
+
+Conflux foundation provided several public RPCs, developer can directly use them interact with Conflux network.
```

Comment:
The Conflux Foundation provides several public RPC endpoints. Developers can use these endpoints directly to interact with the Conflux network.